src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/client/BitbucketServerAPIClient.java

```diff
@@ -684,6 +733,38 @@ public boolean isPrivate() throws IOException {
         return getRepository().isPrivate();
     }
 
+    private <V> List<V> getResources(UriTemplate template, Class<? extends PagedApiResponse<V>> clazz) throws IOException, InterruptedException {
+        template.set("start", 0).set("limit", DEFAULT_PAGE_LIMIT);
+        String url = template.expand();
+
+        try {
```

Comment:
The try scope was also reduced upstream, so please align 😅

Reply:
Ah, I see. I thought that we'd lose the info about the URL in case of a general IOException, but getRequest() wraps that already.